inference-engine/samples/speech_sample/fileutils.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "fileutils.hpp"
+
+void ArkFile::GetFileInfo(const char* fileName, uint32_t numArrayToFindSize, uint32_t* ptrNumArrays, uint32_t* ptrNumMemoryBytes) {
```

Comment:
getFileInfo and LoadFile functions use the same code to read files, can we exclude dupliction? For example by adding the new function OpenFile

Reply:
It is good idea, I will be do it, when I  do refactor of this sample.